ci/packet/packet-cluster.tf.envsubst

```diff
@@ -0,0 +1,62 @@
+module "controller" {
+  source = "../../packet/flatcar-linux/kubernetes"
+
+  providers = {
+    aws      = "aws.default"
+    local    = "local.default"
+    null     = "null.default"
+    template = "template.default"
+    tls      = "tls.default"
+    packet   = "packet.default"
+  }
+
+  dns_zone = "$AWS_DNS_ZONE"
+  dns_zone_id = "$AWS_DNS_ZONE_ID"
+
+  ssh_keys = ["$PUB_KEY"]
+
+  asset_dir = "${pathexpand("~/assets")}"
```

Comment:
Why use different asset directories in aws in packet? If it's for overlap, doesn't it make sense to include the provider in tha path instead?

Reply:
No special meaning, overlap is not a problem because the pipelines run in containers.